packages/http-server/src/http-server.ts

```diff
@@ -92,7 +92,7 @@ export class HttpServer {
     this.server.listen(this.serverOptions);
     await pEvent(this.server, 'listening');
     this._listening = true;
-    this._address = this.server.address();
+    this._address = this.server.address() || '';
```

Comment:
I don't think this is a correct solution, it hides a possible problem - the server address is `null`. [Node.js API docs](https://nodejs.org/dist/latest-v12.x/docs/api/net.html#net_server_address) does not explain the circumstances when `address()` returns `null`, so it's hard to tell when can that happen.

I think our best option is to `assert` that the address is not `null`. 

```ts
const address = this.server.address();
assert(!!address);
this._address  = address!;
```

That way users are immediately notified about the problem.

Reply:
Done in d028673137f